cmake/modules/FindCUDNN.cmake

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+find_path(
+    CUDNN_INCLUDE_DIR cudnn.h
+    HINTS ${CUDNN_DIR} ${CUDA_TOOLKIT_ROOT_DIR}
+    PATH_SUFFIXES include)
+
+find_library(
+    CUDNN_LIBRARY_CUDNN_PATH cudnn
```

Comment:
How about using `CUDNN_LIBRARY_DIR` instead of `CUDNN_LIBRARY_CUDNN`, since there is `CUDNN_INCLUDE_DIR`